NeuralAmpModeler/dsp/get_dsp.cpp

```diff
@@ -16,94 +16,106 @@ void verify_config_version(const std::string version)
     throw std::runtime_error("Unsupported config version");
 }
 
-std::unique_ptr<DSP> get_dsp(const std::filesystem::path dirname)
+std::unique_ptr<DSP> get_dsp(const std::filesystem::path dirname, dspData& returnedConfig)
 {
-  const std::filesystem::path config_filename = dirname / std::filesystem::path("config.json");
```

Comment:
Can you please reformat your PR so that there are 2 spaces per tab? I know the formatting is a bit wild in here (cf #14) but it's difficult to see what's a real change & what's just reformatting.

Reply:
Sure, done! I added .editorconfig file which seems a nice way to do formatting for the project!?

Reply:
Thanks! Much appreciated.

And thanks also for the `.editorconfig`. I haven't worked with this before but I'm sure it'll be just fine :)